lib/process-services-cloud/src/lib/form/components/form-cloud.component.ts

```diff
@@ -131,8 +132,8 @@ export class FormCloudComponent extends FormBaseComponent implements OnChanges,
     nodeId: string;
     formCloudRepresentationJSON: any;
 
-    protected onDestroy$ = new Subject<boolean>();
 
+    destroyRef = inject(DestroyRef);
```

Comment:
I think it can be private readonly (no need to make it public) because it's referred only inside that component